src/libs/antares/study/parts/short-term-storage/properties.cpp

```diff
@@ -119,6 +121,41 @@ bool Properties::loadKey(const IniFile::Property* p)
     return false;
 }
 
+bool Properties::saveToFolder(const std::string& folder) const
+{
+    // Make sure the folder is created
+    if (!Yuni::IO::Directory::Create(folder))
+    {
```

Comment:
Please place this in `AreaList::saveToFolder` (list.cpp) for consistency.

Reply:
Those lines are used in `thermal/cluster_list.cpp`, I agree it's cleaner in AreaList